debian/changelog

```diff
@@ -1,3 +1,10 @@
+argobots (1.2-1) unstable; urgency=medium
+  [ Brian J. Murrell ]
+  * Update to 1.2
+  * Add patch 411e5b3 fixing DAOS-14248
+
+ -- Brian J. Murrell <brian.murrell@intel.com>  Wed, 04 Sep 2024 16:08:44 -0400
```

Comment:
Could you please update the timestamp here and add a comment about the 2nd patch that has been added and it's ticket number.

Please also feel free to change the author here to yourself since you did the most significant amount of work.

Reply:
- [ ] Update author and timestamp

Reply:
> * [x]  Add reference to the new patch [bb0c908](https://github.com/pmodels/argobots/commit/bb0c908abfac4bfe37852eee621930634183c6aa) fixing configuration issue
> * [x]  Update author and timestamp

Fixed with commit b740042